### PR TITLE
enabled is a flag to enable Hazelcast application

### PR DESCRIPTION
**Pull request by jisnardo**
PR for deploy hazelcast & mancenter or individually

question: Is there any way to deploy mancenter alone?

https://github.com/hazelcast/charts/issues/167

Cannot start only management service

https://github.com/hazelcast/charts/issues/161

### CODE REVIEW

**Review by hasancelik — CHANGES_REQUESTED** [review]
@jisnardo,

Could you bump up chart version of `hazelcast` and `hazelcast-enterprise`?

**Comment by hasancelik** [comment]
verify

File: stable/hazelcast-enterprise/templates/metrics-service.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.metrics.enabled }}
+{{- if and (.Values.hazelcast.enabled .Values.metrics.enabled) }}
```
Comment (hasancelik) [inline stable/hazelcast-enterprise/templates/metrics-service.yaml:1]:
```suggestion
{{- if and .Values.hazelcast.enabled .Values.metrics.enabled }}
```
PR builder failed due to below lint error:
```
17:03:43 ==> Linting stable/hazelcast-enterprise
17:03:43 [ERROR] templates/: template: hazelcast-enterprise/templates/metrics-service.yaml:1:19: executing "hazelcast-enterprise/templates/metrics-service.yaml" at <.Values.hazelcast.enabled>: enabled is not a method but has arguments
```
I think above change will make green our PR builder.

Also Could you apply same change to https://github.com/hazelcast/charts/blob/master/stable/hazelcast/templates/metrics-service.yaml?

**Review by hasancelik — CHANGES_REQUESTED** [review]
@jisnardo,

I have added some minor comments, PTAL

**Comment by hasancelik** [comment]
@jisnardo, could you update parameter table at READMEs? 
https://github.com/hazelcast/charts/tree/master/stable/hazelcast#configuration
https://github.com/hazelcast/charts/tree/master/stable/hazelcast-enterprise#configuration

**Comment by hasancelik** [comment]
verify

**Review by hasancelik — APPROVED** [review]
LGTM 👍

**Review by leszko — COMMENTED** [review]
@jisnardo Thanks a lot or the PR 👍 

It looks good. Just one comment, could you add the condition `{{- if .Values.hazelcast.enabled -}}` also to `NOTES.txt`?

**Comment by jisnardo** [comment]
> @jisnardo Thanks a lot or the PR +1
> 
> It looks good. Just one comment, could you add the condition `{{- if .Values.hazelcast.enabled -}}` also to `NOTES.txt`?

Just added.

**Review by leszko — APPROVED** [review]
@jisnardo thanks for the update. LGTM 👍

**Pull request merged**